packages/ui/src/styles/datepicker.scss

```diff
@@ -1,4 +1,4 @@
-@import 'node_modules/flatpickr/dist/flatpickr.min';
+@import '../../node_modules/flatpickr/dist/flatpickr.min';
```

Comment:
I think this will break for when it gets deployed to npm, i assume you did this cause it wasn't working in cypress?

Reply:
yeah, it kept complaining about missing references, then I was fixing them one-by-one

Reply:
I can revert this bit, but then cypress won't have the styling until we find an alternative solution, but that can be done outside of this PR